packages/notebook/src/actions.tsx

```diff
@@ -1966,6 +2000,33 @@ namespace Private {
     activeCell: Cell | null;
   }
 
+  export function isNotebookRendered(
+    notebook: Notebook,
+    translator?: ITranslator
+  ): boolean {
+    translator = translator || nullTranslator;
```

Comment:
You can get the translator from the notebook widget:

```suggestion
  ): boolean {
    translator = notebook.translator;
```